odkx-src/basics-install.rst

```diff
@@ -60,15 +55,9 @@ Installing Services
 Installing the ODK-X Survey App
 -----------------------------------
 
-  1.  From your device's :guilabel:`Settings`, choose :menuselection:`Apps & notifications`. (On older versions of Android, this setting maybe in :menuselection:`Applications` or :menuselection:`Security` depending upon your android version.)
-
-    - Go to :guilabel:`Special app access` in :guilabel:`Advanced` and choose :guilabel:`Install unknown apps`
-    - From the list of applications, select a browser of your choice and check :guilabel:`Allow from this source`.
-    - (On older versions of Android, above two steps are not required, make sure installation from *Unknown Sources* is checked.)
-    
-  2. Open the same web browser that you authorized to install unknown apps on your Android device. (For older versions of Android any web browser can be used since you do not need to specifically authorize the web browser's ability to install.)
-  3. Navigate to https://github.com/odk-x/survey/releases/latest and download the latest ODK-X Survey APK.
-  4. In the download window, you will see ODK_Survey.N.N.apk. - Select it to download the file.
+  1. Open the same web browser that you authorized to install unknown apps on your Android device. (For older versions of Android any web browser can be used since you do not need to specifically authorize the web browser's ability to install.)
```

Comment:
I would mention and link to this step in installing services where you did this so it is clear what is being referred to, and in case anyone tries to do the steps out of order they know what they need to do.